### PR TITLE
Quantized relu to native_functions

### PR DESCRIPTION
**Pull request by z-a-f**
Stack:
&nbsp;&nbsp;&nbsp;&nbsp;:black_circle:&nbsp; **#22316 Quantized relu to native_functions**&nbsp;&nbsp;[:yellow_heart:](https://our.internmc.facebook.com/intern/diff/D16038441/)

Adding the quantized ReLU to the native_functions.yamp, as it has the same signature as non-quantized relu

Differential Revision: [D16038441](https://our.internmc.facebook.com/intern/diff/D16038441/)

### CODE REVIEW

File: test/test_quantized.py [hunk 1]
```diff
@@ -83,18 +81,21 @@ def _pool_output_shape(self, input_size, kernel_size, padding, stride,
...
         qX = torch.quantize_linear(X, scale=scale, zero_point=zero_point,
                                    dtype=torch_type)
-        qY_hat = relu(qX)
+        qY_hat = ops_relu(qX)
+        qY_native_hat = native_relu(qX)
 
         Y[Y < 0] = 0
         qY = torch.quantize_linear(torch.from_numpy(Y), scale=scale, zero_point=zero_point, dtype=torch_type)
-        self.assertEqual(qY.int_repr(), qY_hat.int_repr())
+        self.assertEqual(qY.int_repr(), qY_hat.int_repr(), "torch.ops.quantized.relu failed")
```
Comment (jerryzh168) [inline test/test_quantized.py]: we can compare Quantized Tensor directly now, don't need to do int_repr()

**Comment by jerryzh168** [comment]
Add a test for calling torch.nn.functional.relu?

**Review by raghuramank100 — APPROVED** [review]
Good to go once jerryzh's comments are addressed

File: test/test_quantized.py [hunk 2]
```diff
@@ -85,18 +82,26 @@ def _pool_output_shape(self, input_size, kernel_size, padding, stride,
     @given(Q=qtensor(shapes=array_shapes(1, 5, 1, 5)))
     def test_qrelu(self, Q):
         X, (scale, zero_point), (qmin, qmax), torch_type = Q
-        relu = torch.ops.quantized.relu
+        ops_relu = torch.ops.quantized.relu
```
Comment (jerryzh168) [inline test/test_quantized.py]: remove?

**Review by jerryzh168 — APPROVED** [review]

**Pull request closed without merging**